cpp/models/abm/migration_rules.cpp

```diff
@@ -54,8 +54,8 @@ LocationType go_to_school(const Person& person, TimePoint t, TimeSpan dt, const
 
     if (current_loc == LocationType::Home && t < params.get<LockdownDate>() && t.day_of_week() < 5 &&
         person.get_go_to_school_time(params) >= t.time_since_midnight() &&
-        person.get_go_to_school_time(params) < t.time_since_midnight() + dt &&
-        person.get_age() == AgeGroup::Age5to14 && person.goes_to_school(t, params) && !person.is_in_quarantine()) {
+        person.get_go_to_school_time(params) < t.time_since_midnight() + dt && person.get_age() == AgeGroup(1) &&
```

Comment:
This can't be hardcoded.

Reply:
As per suggested, I add the parameters AgeGroupGotoSchool and AgeGroupGotoWork in MigrationParameters so the user can specify them in the initialisation of the model.